exporter/collector/integrationtest/testdata/fixtures/workload_metrics_expect.json

```diff
@@ -6326,11 +6296,7 @@
               }
             }
           ]
-        }
-      ]
-    },
-    {
```

Comment:
note: because we dropped one of the metrics above, the batch cutoff is now one point later.